internal/cloudfront/service_helpers.go

```diff
@@ -37,10 +37,11 @@ func newDistribution(ingresses []k8s.CDNIngress, group, webACLARN, distARN strin
 		cfg.CloudFrontPriceClass,
 		group,
 		cfg.CloudFrontWAFARN,
+		cfg,
```

Comment:
nit: since the Config is an argument now, we can probably remove many arguments of `NewDistributionBuilder`. If you look closely in the lines above, many of them take fields from cfg.

Reply:
Thanks for pointing this out, ended up removing all the input fields bar `group` and the config, much cleaner now!